octobot_trading/modes/script_keywords/context_management.py

```diff
@@ -492,10 +495,11 @@ async def backtesting_results(self, with_lock=False, cache_size=None, database_a
         display = commons_display.display_translator_factory()
         run_dbs_identifier = databases.RunDatabasesIdentifier(
             self.trading_mode_class,
-            self.optimization_campaign_name or optimization_campaign.OptimizationCampaign.get_campaign_name(),
```

Comment:
why removing this ?
Except for that, I think we can merge

Reply:
I added it again and adapted it for live as well

Reply:
great 👍